# Grasp

string id
string method

uint8 index
float32 estimated_reward

geometry_msgs/Pose pose
float32 gripper_width

bool safe